### PR TITLE
updated references and generated new docs

### PR DESCRIPTION
**Pull request by StephenCoady**
small PR to update references to new cordova PR. once merge https://github.com/feedhenry/mobile-security-cordova-template/pull/2 can be merged.

### CODE REVIEW

**Comment by StephenCoady** [comment]
ping @wei-lee @TommyJ1994

File: docs/mobileAppSelfDefenceOverview.adoc [hunk 1]
```diff
@@ -248,16 +248,7 @@ Detecting if the application is built in debug mode.
...
-....
-
-==== Detecting Latest OS
-Detecting if the application is running the latest version of Android or iOS.
-
-.Code
-[source,javascript,linenums,indent=0]
-....
-include::https://raw.githubusercontent.com/feedhenry/mobile-security-cordova-template/master/src/pages/deviceTrust/deviceTrust.ts[tags=detectLatestOS]
+include::https://raw.githubusercontent.com/aerogear/cordova-showcase-template/master/src/pages/security/deviceTrust/deviceTrust.ts[tags=detectDebug]
```
Comment (wei-lee) [inline docs/mobileAppSelfDefenceOverview.adoc:251]: are the tags still there in the showcase app?

**Review by wei-lee — APPROVED** [review]
LGTM.

File: docs/mobileAppSelfDefenceOverview.adoc (hunk 1 above)
Comment (StephenCoady) [reply docs/mobileAppSelfDefenceOverview.adoc:251]: yes still there

**Comment by StephenCoady** [comment]
@wei-lee I can't merge on this repo for some reason. could you do it when you get a chance.

**Pull request merged**